resources/test_data/sqlite_testrunner_queries.sql

```diff
@@ -105,9 +105,9 @@ SELECT a AS x, SUM(b) FROM mixed GROUP BY x HAVING x > 10;
 SELECT a AS x, SUM(b) FROM mixed GROUP BY x HAVING x > 10;
 SELECT a AS a1, a AS a2 FROM mixed;
 SELECT a AS a1, b AS b2, b AS b3, a AS a3, b AS b1, a AS a2 FROM mixed;
-SELECT COUNT(*) AS cnt1, COUNT(*) AS cnt2, COUNT(*) AS cnt3 FROM mixed GROUP BY a;
```

Comment:
Feel free to add this as an alternative. I'd still leave the old version intact here.

Reply:
The original reason for the change was to just revert all changes from the workaround: fb5f675be6b9e869fd2308b3f6199e51f5da63b5

Reply:
I see. In any case, there is some value in checking that even that somewhat non-sensical query works.